applicatie/static/js/custom/bulk.js

```diff
@@ -147,7 +242,7 @@ function form_control_reset_rij(corr) {
     foo = corr.find("#bedrag");
```

Comment:
niet jouw code, maar misschien iets kleins: waarom heet de variabele foo, misschien een zinvollere naam geven?

Reply:
het had ook gewoon x kunnen zijn. https://en.wikipedia.org/wiki/Foobar

Reply:
ik snap waar het vandaan komt, het is gewoon geen goede naam, x ook niet, er wordt een bedrag opgehaald en daarna gebeuren er nog wat dingen met foo, het is een kleine moeite om een geschiktere naam te bedenken dan foo of x, en bevordert de leesbaarheid. Maar goed, niet per se in de PR, dus wat mij betreft laat je het zo.

Reply:
Eens, maar ik heb/had even geen inspiratie voor een betere naam :) Kunnen we later even aanpassen.